content/reference/drops/search_drop.md

```diff
@@ -0,0 +1,20 @@
+# Search Drop
+
+Return search for rentals.
+
+#### Example
+
+~~~ liquid
+  {% for rental in search.rentals %}
+    <div class="col-sm-4">
+      {% include 'snippets/rental_preview' %}
+    </div>
+  {% endfor %}
+~~~
+
+Available attributes:
+
+* display_type
+* display_url
```

Comment:
`display_url` is a private method, so it won't probably be available here, but `display_list_url`, `display_photos_url` and `display_map_url` would be, there are the ones defined with meta magic.